### DISCUSSION TITLE
Threshold percent of mean

### DISCUSSION THREAD

**Discussion by philomory** [grafana/grafana#38214] [Dashboards, Panels & Visualizations]
**What would you like to be added**: I would like to be able to set percentage thresholds to be the percentage of the mean, rather than the percentage of the maximum (or, percentage of the way it is between minimum and maximum).

**Why is this needed**:

I'd like to be able to have a gauge defined that is colored green if the current value is at least 90% of the average value for the data points available. If I set it to 90% currently, then one over-large data point will cause the threshold to fail to be met for as long as that data point remains in the relevant time window.

I need to use a percentage because I'm using a repeated panel with the same measurement across different locations, and each location (of which we have nearly a hundred) has a different typical value.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
